tests/netflix/test_ddb.py

```diff
@@ -0,0 +1,314 @@
+from service_capacity_modeling.interface import AccessConsistency
+from service_capacity_modeling.interface import CapacityDesires
+from service_capacity_modeling.interface import certain_int
+from service_capacity_modeling.interface import Consistency
+from service_capacity_modeling.interface import DataShape
+from service_capacity_modeling.interface import GlobalConsistency
+from service_capacity_modeling.interface import Interval
+from service_capacity_modeling.interface import QueryPattern
+
+storage_only_desires = CapacityDesires(
+    service_tier=0,
+    query_pattern=QueryPattern(
+        access_consistency=GlobalConsistency(
+            same_region=Consistency(
+                target_consistency=AccessConsistency.read_your_writes
+            )
+        ),
+        estimated_read_per_second=certain_int(0),
+        estimated_write_per_second=certain_int(0),
+    ),
+    data_shape=DataShape(
+        estimated_state_size_gib=Interval(low=10, mid=100, high=1000, confidence=0.98),
+    ),
+)
+
+read_desires = CapacityDesires(
+    service_tier=0,
+    query_pattern=QueryPattern(
+        access_consistency=GlobalConsistency(
+            same_region=Consistency(target_consistency=AccessConsistency.eventual)
+        ),
+        estimated_read_per_second=Interval(
+            low=100, mid=1000, high=10000, confidence=0.98
+        ),
+        estimated_write_per_second=certain_int(0),
+    ),
+    data_shape=DataShape(
+        estimated_state_size_gib=Interval(low=10, mid=100, high=1000, confidence=0.98),
+    ),
+)
+
+write_desires = CapacityDesires(
+    service_tier=0,
+    query_pattern=QueryPattern(
+        estimated_write_per_second=Interval(
+            low=100, mid=1000, high=10000, confidence=0.98
+        )
+    ),
+    data_shape=DataShape(
+        estimated_state_size_gib=Interval(low=10, mid=100, high=1000, confidence=0.98),
+    ),
+)
+
+
+def test_plan_storage():
+    result = planner.plan(
+        model_name="org.netflix.ddb",
+        region="us-east-1",
+        desires=storage_only_desires,
+        extra_model_arguments={"number_of_regions": 1},
+    )
+    mean_plan = result.mean
+    assert len(mean_plan) > 0
+    assert mean_plan[0].requirements.regional[0].disk_gib == certain_int(100)
+    regional_req = {
+        "read_capacity_units": 0,
+        "write_capacity_units": 0,
+        "data_transfer_gib": 0.0,
+        "replicated_write_capacity_units": 0,
+    }
+    assert mean_plan[0].requirements.regional[0].context == regional_req
+    annual_costs = {
+        "dynamo.regional-writes": Decimal("0.0"),
+        "dynamo.regional-reads": Decimal("0.0"),
+        "dynamo.regional-storage": Decimal("300.0"),
+        "dynamo.regional-transfer": Decimal("0.0"),
+        "dynamo.data-backup": Decimal("240.0"),
+    }
+    assert mean_plan[0].candidate_clusters.annual_costs == annual_costs
+    assert round(mean_plan[0].candidate_clusters.total_annual_cost, 0) == round(540, 0)
```

Comment:
If we're going to make exact assertions like this you probably want to use plan_certain instead of plan since the latter still will simulate the inputs.

Also I think range assertions are probably more robust longer term (e.g. in this case >100 and < 1000).

Same advice for the rest of the file, `plan` is non deterministic (but it is reproducible) within a range of valid outputs. `plan_certain` has no simulation.